ops/index.js

```diff
@@ -64,21 +63,14 @@ async function migrate(web3) {
     gas: (await web3.eth.getBlock("latest")).gasLimit - 100000
   };
 
-  const GEN = new web3.eth.Contract(DAOToken.abi, undefined, opts);
-  const gen = await GEN.deploy({
-    data: DAOToken.bytecode,
-    arguments: ["GEN", "GEN", "1000000000000000000000000000"]
-  }).send();
-
-  const GP = new web3.eth.Contract(GenesisProtocol.abi, undefined, opts);
-  const gp = await GP.deploy({
-    data: GenesisProtocol.bytecode,
-    arguments: [gen.options.address]
+  const Rep = new web3.eth.Contract(Reputation.abi, undefined, opts);
+  const rep = await Rep.deploy({
```

Comment:
why deploy ?

Reply:
This is the migration script

Reply:
are we migrating here ? not sure .
If this is for testing purposes , so move it under test folder

Reply:
this is the `migrate` function in `index.js` that is run before the tests. Not sure we need to move it to the `test` folder.